docs/shared/recipes/add-stack/add-lit.md

```diff
@@ -0,0 +1,206 @@
+# Add a New Lit Project
+
+The code for this example is available on Github:
+
+{% github-repository url="https://github.com/nrwl/nx-recipes/tree/main/lit" /%}
+
+**Supported Features**
+
+Because we are not using an Nx plugin for Lit, there are few items we'll have to configure manually. We'll have to configure our own build system. There are no pre-created Lit-specific code generators. And we'll have to take care of updating any framework dependencies as needed.
+
+{% pill url="/core-features/run-tasks" %}✅ Run Tasks{% /pill %}
+{% pill url="/core-features/cache-task-results" %}✅ Cache Task Results{% /pill %}
+{% pill url="/core-features/share-your-cache" %}✅ Share Your Cache{% /pill %}
+{% pill url="/core-features/explore-graph" %}✅ Explore the Graph{% /pill %}
+{% pill url="/core-features/distribute-task-execution" %}✅ Distribute Task Execution{% /pill %}
+{% pill url="/core-features/integrate-with-editors" %}✅ Integrate with Editors{% /pill %}
+{% pill url="/core-features/automate-updating-dependencies" %}✅ Automate Updating Nx{% /pill %}
+{% pill url="/core-features/enforce-project-boundaries" %}✅ Enforce Project Boundaries{% /pill %}
+{% pill url="/core-features/plugin-features/use-task-executors" %}🚫 Use Task Executors{% /pill %}
+{% pill url="/core-features/plugin-features/use-code-generators" %}🚫 Use Code Generators{% /pill %}
+{% pill url="/core-features/automate-updating-dependencies" %}🚫 Automate Updating Framework Dependencies{% /pill %}
+
+## Install Lit and Other Dependencies
+
+Install all the dependencies we need:
+
+{% tabs %}
+{%tab label="npm"%}
+
+```shell
+npm i --save-dev @nx/node lit http-server
```

Comment:
why do we need `@nx/node`? Doesn't it  just use `@nx/esbuild`?

Reply:
I'm using the `@nx/node:app` generator to scaffold out the project.  `@nx/esbuild` doesn't have a generator.

I intentionally went that route instead of building up a minimally set up from scratch, because we want to give people simple instructions that will set them up for success.  The goal for these recipes is not to show them the bare minimum files that are needed to get Nx to work with their framework.

`@nx/node:app` gives them unit test and e2e test scaffolding out of the box.